src/app/utils/persistentStorageTemplate.ts

```diff
@@ -0,0 +1,54 @@
+export const fileStore = `"file-store": {
+  "shared": false,
+  "data": {
+    "path": "data"
+  },
+  "index": {
+    "path": "index"
+  },
+  "write-behind": {
+    "modification-queue-size": "2048"
+  }
+}`;
+
+export const remoteStore = `"remote-store": {
+  "cache": "mycache",
+  "raw-values": "true",
+  "remote-server": [
+    {
+      "host": "one",
+      "port": "12111"
+    },
+    {
+      "host": "two"
+    }
+  ],
+  "connection-pool": {
+    "max-active": "10",
+    "exhausted-action": "CREATE_NEW"
+  }
+}`;
+
+export const sqlStore = `"table-jdbc-store": {
+  "dialect": "H2",
+  "shared": "true",
+  "table-name": "books",
+  "schema": {
+    "message-name": "books_value",
+    "package": "library"
+  }
+}`;
```

Comment:
I think we need to include the "connection-pool" here to indicate to the user that this must be configured.

```json
	"table-jdbc-store": {
		"dialect": "H2",
		"shared": "true",
		"table-name": "books",
		"connection-pool": {
			"connection-url": "jdbc:h2:mem:infinispan",
			"driver": "org.h2.Driver",
			"username": "sa",
			"password": "changeme"
		}
	}
```